doc/concepts/configuration/configuration_connections.rst

```diff
@@ -0,0 +1,292 @@
+
+
+.. _configuration_connections_listen_uri:
+
+Listen URI
+----------
+
+To configure URIs used to listen for incoming requests, use the :ref:`iproto.listen <configuration_reference_iproto_listen>` configuration option.
+
+.. _configuration_connections_listen_address:
+
+One listen address
+~~~~~~~~~~~~~~~~~~
+
+The example below shows how to set a listening IP address for ``instance001`` to ``127.0.0.1:3301``:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/iproto_listen_address/config.yaml
+    :start-at: instance001
+    :end-at: '127.0.0.1:3301'
+    :language: yaml
+    :dedent:
+
+.. _configuration_connections_listen_addresses:
+
+Multiple listen addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example, ``instance001`` listens on two IP addresses:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/iproto_listen_addresses/config.yaml
+    :start-at: instance001
+    :end-at: '127.0.0.1:3302'
+    :language: yaml
+    :dedent:
+
+.. _configuration_connections_listen_port:
+
+Listen port
+~~~~~~~~~~~
+
+You can pass only a port value to ``iproto.listen``:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/iproto_listen_port/config.yaml
+    :start-at: instance001
+    :end-at: '3301'
+    :language: yaml
+    :dedent:
+
+In this case, this port is used for all IP addresses the server listens on.
+
+
+.. _configuration_connections_listen_ssl:
+
+SSL parameters
+~~~~~~~~~~~~~~
+
+You can enable :ref:`traffic encryption <configuration_connections_ssl>` for a connection using the ``params`` section of the specified URI:
+
+..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/ssl_without_ca/config.yaml
+    :language: yaml
+    :start-at: instance001:
+    :end-before: instance002:
+    :dedent:
+
+
+.. _configuration_connections_unix_socket:
+
+Unix domain socket
+~~~~~~~~~~~~~~~~~~
+
+For local development, you can enable communication between cluster members by using Unix domain sockets:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/iproto_listen_socket/config.yaml
+    :start-at: instance001
+    :end-at: tarantool.iproto
+    :language: yaml
+    :dedent:
+
+
+
+
+.. _configuration_connections_advertise:
+
+Advertise settings
+------------------
+
+Advertise settings (:ref:`iproto.advertise.* <configuration_reference_iproto_advertise>`) let other cluster members or clients know how to connect to the current Tarantool instance.
+These settings might include the credentials required to connect to this instance, a URI used to listen for incoming requests, and SSL settings.
+
+If an advertise URI is not specified explicitly, a :ref:`listen URI <configuration_connections_listen_uri>` of this instance is used.
+In this case, you need at least to specify credentials for connecting to this instance.
+
+..  NOTE::
+
+    If several listen URIs are specified, the first one is used as an advertise URI.
```

Comment:
Strictly speaking, first one *that is suitable to connect*. It excludes URIs with `0.0.0.0` or `[::0]` IP address and URIs with zero TCP/IP port.